README.md

```diff
@@ -72,7 +72,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- Live Demo Link (N/A) coming soon.
+- N/A
```

Comment:
- [Optional] Dear @dd-obua, good job making the changes as requested by the previous reviewers. But for the live demo section, It's required to have a working link or no live demo section. Please consider removing the live demo section unless your project has a live demo.